watertap/examples/flowsheets/full_treatment_train/analysis/tests/test_multi_sweep.py

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import pytest
+import os
+from pyomo.environ import value
+from watertap.examples.flowsheets.full_treatment_train.analysis import (flowsheet_NF,
+                                                               flowsheet_NF_no_bypass,
+                                                               flowsheet_single_stage,
+                                                               flowsheet_two_stage,
+                                                               flowsheet_NF_two_stage,
+                                                               flowsheet_softening,
+                                                               flowsheet_softening_two_stage)
+from watertap.examples.flowsheets.full_treatment_train.analysis.multi_sweep import *
+from watertap.tools.parameter_sweep import _init_mpi, LinearSample, parameter_sweep
+
+@pytest.mark.component
```

Comment:
We should be using[`pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html) here instead of the `for` loop below.

Reply:
agreed, fixed!